src/FixedDataTableBufferedRows.js

```diff
@@ -142,7 +144,7 @@ var FixedDataTableBufferedRows = React.createClass({
 
       this._staticRowArray[i] =
         <FixedDataTableRow
-          key={i}
+          key={rowKeyGetter(rowIndex)}
```

Comment:
@conorbranagan Sorry for the delayed response, got busy with some work and forgot about this PR.
Can we just do something like 

   var rowKey = props.rowKeyGetter ? rowKeyGetter(rowIndex) : i

passing array index seems wierd.